tests/features/local_translations.feature

```diff
@@ -21,9 +23,17 @@ Feature: Local translations
     When I click "Translate locally" in the "Bulgarian" row
     Then the translation form element for the "Title" field should contain "My title"
     And the translation form element for the "Body" field should contain "My body"
+    And the translation form element for the "Link Uri (1)" field should contain "https://example.com"
+    And the translation form element for the "Link Title (1)" field should contain "Example"
+    And the translation form element for the "Link Uri (2)" field should contain "/node"
+    And the translation form element for the "Link Title (2)" field should contain "Node"
```

Comment:
I kinda agree with @dxvargas here, why not create a different step "Given a translatable node with the "My title" title and "My body" body and the links:" that receives a table of link-title pairs? It would make it more readable.